devolving_music/models/song_submission.py

```diff
@@ -1,13 +1,32 @@
+from typing import Iterable
 from django.db import models
 from .event import Event
 from .song import Song
-
-
+from devolving_music.lib.elo_scoring import elo_rating
+from django.utils.functional import cached_property
+from bisect import bisect_left
 class SongSubmission(models.Model):
```

Comment:
+2 newlines before class definition.

Figure out how to get VSCode to give PEP 8 formatting hints, it seems like pylint doesn't do as much as I thought

Reply:
this seems like a lot of work for very little benefit